docs/content/quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Quickstart Guide
+descriptions: Get started using Porter
+---
+
+## Getting Porter
+
+First make sure Porter is installed.
+Please see the [installation instructions](./install.md) for more info.
+
+## Creating a new installer
+Use the `porter create` command to start a new project:
+```
+mkdir -p my-installer/ && cd my-installer/
+porter create
+```
+
+This will create a file called `porter.yaml` which contains the configuration for your installer. Modify and customize this file for your application's needs.
+
+Here is a very basic `porter.yaml` example:
+```
+name: my-installer
+version: 0.1.0
+description: "this application is extremely important"
+invocationImage: my-dockerhub-user/my-installer:latest
+mixins:
+  - exec
+install:
+  - description: "Install Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - echo Hello World
+uninstall:
+  - description: "Uninstall Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - echo Goodbye World
+```
+
+## Building a CNAB bundle
```

Comment:
nit: Let's change this to "Build the Bundle". I try to not reference CNAB for the most part unless it is necessary.